style.css

```diff
@@ -0,0 +1,40 @@
+* {
+  margin: 0;
+}
+.white { color: white; }
+
+.header-container {
+  display: flex;
+  width: auto;
+  height: auto;
+  align-items: center;
+  background-color: #3c3a39;
+  justify-content: space-between;
+  padding: 20px;
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible, The typos (`font`), spacing(`margin, padding`), and the color of your element should be similar to the Figma template 
`Hint: To see the styles applied to a give section on the page, just that section or text in Figma and on the right side you will have them popup` :smile: 
- [ ] Also make sure you have the social icons and that while background image on your page as is the case in Figma.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205025547-7d5d4d7f-f3ce-43c2-8b3e-a547f9b434c3.png)
The Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205025959-3712497f-a47b-4ebd-915d-b84500dfa4b3.png)

Reply:
<img width="181" alt="mobile" src="https://user-images.githubusercontent.com/87595063/205086827-f82c41e5-079b-4c29-89d7-3e0e5c9c6c76.png">
Thank you for your comment, but I don't know why you don't see my social media icons while I put them.

Reply:
@christianbiring1

Reply:
Hello, @christianbiring1 am trying to set my local font to the text but the linter displays errors what should I do if I want to use the same font as it's on the Figma design?